macosx/PrefsController.m

```diff
@@ -1497,7 +1484,7 @@ - (void) updateGrowlButton
         [fGrowlAppButton setHidden: NO];
 
 #warning remove NO
-        [fGrowlAppButton setEnabled: NO && [GrowlApplicationBridge isGrowlURLSchemeAvailable]];
+        [fGrowlAppButton setEnabled: NO /*&& [GrowlApplicationBridge isGrowlURLSchemeAvailable] */];
```

Comment:
This line is not executed anyway, but it would not compile under ARC since there is no such selector declared in GrowlApplicationBridge.h. This could be circumvented by calling this selector using its name as a string, but I am not sure why the method is not available in the first place. The string workaround also does not work with another missing method that takes BOOL as a parameter.